skcosmo/selection/FPS.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+from skcosmo.pcovr.pcovr_distances import get_Ct, get_Kt
+
+
+def _calculate_distances_(points, ref_idx, idxs=None):
+    if idxs is None:
+        idxs = range(points.shape[0])
+    return np.array(
```

Comment:
I'm not sure creating a list using list comprehension and then converting it to a numpy array is the most efficient thing to do. It should be fine for now, but if there is a way to transform this to using only numpy operations it would be nice